### DISCUSSION TITLE
OpenSSL CLA email bounces

### DISCUSSION THREAD

**Discussion by danielgustafsson** [openssl/openssl#24795] [Q&A]
When trying to email my CLA to the designated email address I get it bouncing back:
```
<[cla@openssl.org](mailto:cla@openssl.org)>: host [mta.openssl.org](http://mta.openssl.org/)[34.32.178.11] said: 550 5.1.1
   <[cla@openssl.org](mailto:cla@openssl.org)>: Recipient address rejected: User unknown in virtual
   alias table (in reply to RCPT TO command)
```
Any assistance on how to resolve that, I have a few patches that I'd like to PR.

**Comment by nhorman** [comment]
@quarckster can you please look into this?

**Comment by mattcaswell** [comment]
@danielgustafsson - in the meantime while this is fixed please email legal@opensslfoundation.org

**Comment by t8m** [comment]
This is fixed now.

**Comment by danielgustafsson** [comment]
Emailed `cla@` today and so far no bounce, thanks for the assistance!
